runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -154,11 +148,10 @@ public Token appendCheckpoint(Token snapshotTimestamp) {
                 .snapshot(snapshotTimestamp)
                 .build()
                 .begin();
-        try (Timer.Context context = MetricsUtils.getConditionalContext(appendCheckpointTimer)) {
+        try (Stream<Map.Entry> entries = this.map.entryStream()) {
             // A checkpoint writer will do two accesses one to obtain the object
             // vlo version and to get a shallow copy of the entry set
             log.info("appendCheckpoint: Started checkpoint for {} at snapshot {}", streamId, snapshotTimestamp);
```

Comment:
With this change, shall we put this log info before the try-with statement?

Reply:
done.